snap/snapcraft.yaml

```diff
@@ -54,7 +54,8 @@ parts:
     source-type: git
     plugin: nil
     override-build: |
-      (cd vendor && rm -rf * && git clone https://github.com/sergio-costas/ubuntu-desktop-provision.git && cd ubuntu-desktop-provision && git checkout add-login-on-end && cd packages/ubuntu_wizard && flutter pub get)
+      (mv vendor/ubuntu-desktop-provision/packages/subiquity_client . && cd vendor && rm -rf * && git clone https://github.com/sergio-costas/ubuntu-desktop-provision.git && cd ubuntu-desktop-provision && git checkout add-login-on-end && cd packages/ubuntu_wizard && flutter pub get)
+      (rm -rf vendor/ubuntu-desktop-provision/packages/subiquity_client && mv subiquity_client vendor/ubuntu-desktop-provision/packages/)
```

Comment:
I would put both pieces of code in the same parentheses, just to keep everything in the same place.

Reply:
That was becoming a bit long, this was a bit more readable.